doc/source/tutorials/quantum_gravity/spacetime_classification.ipynb

```diff
@@ -7,6 +7,8 @@
    "source": [
     "## Classifying discrete spacetimes by dimension\n",
     "\n",
+    "Note: Breaks on M1 Macs because of `tensorflow` causing kernel panic.\n",
```

Comment:
can you also add this to Known Issues in the README?

Reply:
Added to Known Issues in README.